src/pages/Staking/index.tsx

```diff
@@ -106,7 +107,15 @@ const Staking = (): JSX.Element => {
   const { t } = useTranslation();
   const prices = useGetPrices();
 
-  const { governanceTokenBalance, bridgeLoaded, address } = useSelector((state: StoreType) => state.general);
+  const { bridgeLoaded, address } = useSelector((state: StoreType) => state.general);
+
+  const {
+    isIdle: governanceTokenBalanceIdle,
+    isLoading: governanceTokenBalanceLoading,
+    data: governanceTokenBalance,
+    error: governanceTokenBalanceError
+  } = useGovernanceTokenBalance();
+  useErrorHandler(governanceTokenBalanceError);
```

Comment:
Hi @crypto-engineer as discussed, it would be great to move `useErrorHandler` into the hook so that we only need to call it once in the hook, rather than in each component.

Reply:
Done!